R/grob-null.r

```diff
@@ -4,8 +4,8 @@
 #' @export
 zeroGrob <- function() .zeroGrob
 
-# Will get assigned in .onLoad()
 .zeroGrob <- NULL
+on_load(.zeroGrob <- grob(cl = "zeroGrob", name = "NULL"))
```

Comment:
Does this need to be `<<-`?

Reply:
No. With `on_load()` the expression is evaluated in the package namespace directly